aws/build.rake

```diff
@@ -94,7 +95,7 @@ def threaded_each(array, thread_count=2)
   threads.each(&:join)
 end
 
-if (rack_env?(:staging) && CDO.name == 'staging') || rack_env?(:development)
+if rack_env?(:staging) || rack_env?(:development)
```

Comment:
Yes, if it's not required, it seems like this optimization should be in a different PR to reduce the risk and change surface.    

It seems possible that CDO.name != rack_env, at least in principle.